distributed/s3.py

```diff
@@ -0,0 +1,31 @@
+import boto3
+from dask.imperative import Value
+from distributed import default_executor
+
+DEFAULT_PAGE_LENGTH = 1000
+
+
+def get_list_of_summary_objects(bucket_name, prefix='', delimiter='', page_size=DEFAULT_PAGE_LENGTH):
+    s3 = boto3.resource('s3')
+    return list(s3.Bucket(bucket_name).objects.filter(Prefix=prefix, Delimiter=delimiter).page_size(page_size))
```

Comment:
This makes sense. Perhaps this should be renamed to 'response_keys_delimiter'? I imagine that if users do pd.read_csv to convert bytes, the delimiter there is totally a  different thing and yield confusing code- Better to avoid overloaded kwargs.